packages/vaadin-crud/src/vaadin-crud.js

```diff
@@ -460,6 +464,12 @@ class CrudElement extends ElementMixin(ThemableMixin(PolymerElement)) {
         notify: true
       },
 
+      hideToolbar: {
```

Comment:
1. Please add JSDoc annotation to this property.
2. Please add the property to `vaadin-crud.d.ts` and use `@attr {boolean} hide-toolbar` there.

Reply:
Thanks for the suggestion, I made the change and will apply it accordingly in flow part too.